### DISCUSSION TITLE
Any way to move menu items in side-bar nav that have nested pages to be listed first?

### DISCUSSION THREAD

**Discussion by Robinaite** [squidfunk/mkdocs-material#3837] [Q&A] [answered]
Hey all! 

I am not sure how these items are called, but as shown in the screenshot below, is there any way for the items to render the other way around? The menu items with nested pages first and single pages after?

![image](https://user-images.githubusercontent.com/6293634/164980086-912a4315-db77-4ec1-9821-90266c390e93.png)

Basically having it like this: 
![image](https://user-images.githubusercontent.com/6293634/164980191-d051cfb5-cddf-4077-b2c4-70c15ca08985.png)
(This was me moving the items in the inspector)

Thanks!

**Comment by squidfunk** [comment, marked as answer]
Yes, you can arrange the pages as you like in `mkdocs.yml`. Material for MkDocs will render the pages exactly as specified in the [`nav`](https://www.mkdocs.org/user-guide/configuration/#nav) configuration setting.
